### ISSUE TITLE
Remove ShutUp10

### ISSUE THREAD

**Issue by nemchik** [LeDragoX/Win-Debloat-Tools#167]
### Description

ShutUp10 seems great as a tool to use manually, maybe after running this script, but it's insanely unintuitive to try and decipher what all the config options in https://github.com/LeDragoX/Win-Debloat-Tools/blob/main/src/configs/shutup10/ooshutup10.cfg and https://github.com/LeDragoX/Win-Debloat-Tools/blob/main/src/configs/shutup10/ooshutup10-default.cfg are.

There also seem to be conflicting configurations between ShutUp10 and your personal tweaks script.
https://github.com/LeDragoX/Win-Debloat-Tools/blob/d43f8b7511041d2db60d9d79b40034e1c2be45a7/src/scripts/Register-PersonalTweaksList.ps1#L236-L242
This should allow these two settings to be user configurable
![image](https://github.com/LeDragoX/Win-Debloat-Tools/assets/725456/edd578c1-226e-480c-bbc8-e7bdebacf149)
However since the config has these two lines
https://github.com/LeDragoX/Win-Debloat-Tools/blob/d43f8b7511041d2db60d9d79b40034e1c2be45a7/src/configs/shutup10/ooshutup10.cfg#L115 and https://github.com/LeDragoX/Win-Debloat-Tools/blob/d43f8b7511041d2db60d9d79b40034e1c2be45a7/src/configs/shutup10/ooshutup10.cfg#L117
These apply the settings to HKLM while your tweaks only remove the entries in HKCU.

Just for visibility, you also have https://github.com/LeDragoX/Win-Debloat-Tools/blob/d43f8b7511041d2db60d9d79b40034e1c2be45a7/src/configs/shutup10/ooshutup10.cfg#L114 and https://github.com/LeDragoX/Win-Debloat-Tools/blob/d43f8b7511041d2db60d9d79b40034e1c2be45a7/src/configs/shutup10/ooshutup10.cfg#L116 which are the HKCU entries that your tweaks script later removes.

As you might have guessed, deciphering which config options in ShutUp10 are which was a bit of a chore, and it's much easier to look through the rest of your script to see exactly what's going on.

I don't know if there are things in ShutUp10 that you really like that would be a pain to implement directly in your scripts, but if not I personally think it would be great to remove ShutUp10 entirely and implement anything it was covering (if your scripts don't already do so).

### Reason

Explanation provided above regarding reasoning.
tldr; ShutUp10 config is cryptic and it's not easy to see what the config does compared to almost everything else in the script being very easy to see what's being changed

**Comment by entauri** [comment]
Partially related, but if I'm already using oo shutup (and simplewall) before running this script do I have to restore everything default?

**Comment by LeDragoX** [comment]
@nemchik sorry for the delay, so, the script started without many tweaks and have expanded a lot since then, and when I updated the shutup10 with their recommended preset, it messed up with some settings I disliked, like the Microsoft Edge searching history not appearing, I did put that "glue", but I don't know if it's already the time to remove it, and let the script do every tweak.

They also update a lot, containing tweaks for Copilot and every new stuff Microsoft toss into Windows. Maybe it's really time to remove Shutup10, but I need to include the newer tweaks first...
